packages/app/src/app/pages/Dashboard/Sidebar/BottomMessages/TrialExpiring.tsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import { Link as RouterLink } from 'react-router-dom';
+import { Stack, Text, Link } from '@codesandbox/components';
+
+export const TrialExpiring: React.FC<{
+  daysLeft: number;
+  isAdmin: boolean;
+}> = ({ daysLeft, isAdmin }) => (
+  <Stack align="flex-start" direction="vertical" gap={2}>
+    <Text css={{ color: '#c2c2c2', fontWeight: 500, fontSize: 12 }}>
+      {daysLeft === 0 && <>Your trial expires today.</>}
+      {daysLeft === 1 && <>Your trial expires tomorrow.</>}
+      {daysLeft > 1 && <>{daysLeft} days left on your trial.</>}
+    </Text>
+    <Text css={{ color: '#999', fontWeight: 400, fontSize: 12 }}>
+      After this period, your Team Pro subscription will be automatically
+      renewed.
+    </Text>
+    {isAdmin && (
+      <Link
+        as={RouterLink}
+        to="/dashboard/settings"
```

Comment:
Should this link to this screen?

<img width="1680" alt="Screen Shot 2022-10-30 at 22 28 36" src="https://user-images.githubusercontent.com/24959348/199761744-93dc60ac-82eb-4376-944a-5b9da4ac2b2e.png">